setup/environment.yml

```diff
@@ -88,13 +88,13 @@ dependencies:
     - eth-typing==2.1.0
     - eth-utils==1.4.1
     - hexbytes==0.1.0
-    - hummingsim==20190508.1
+    - hummingsim==20190509
```

Comment:
I've compiled a newer package version for today, `20190509.1`.

Also, you should remember to change `environment-linux.yml` as well.